test/README.md

```diff
@@ -79,9 +79,8 @@ To run all legacy Vim tests:
 
 To run a *single* legacy test set `TEST_FILE`, for example:
 
-    TEST_FILE=test_syntax.res make oldtest
+    TEST_FILE=test_syntax make oldtest
```

Comment:
if `TEST_FILE=test_syntax.vim` works then it should recommend that. This clarifies for new contributors the separation between  "vimscript tests" and  "lua tests".